webapp/client/src/components/FormHeader.js

```diff
@@ -4,7 +4,7 @@ import FormHeaderIntro from "./FormHeaderIntro";
 export default function FormHeader({ title, intro, hideIntro }) {
   const paragraphs = [].concat(intro); // intro could be an array or a single string/element/object. This will give us an array in both situations.
   return (
-    <div>
+    <div className="section-intro">
```

Comment:
on our react pages there was a section-intro missing... because of that we duplicated styles

Reply:
so we have to make sure we have section-intro all all react pages. So the top styling is unified.